src/Rules/FunctionCallParametersCheck.php

```diff
@@ -61,7 +60,7 @@ public function check(
 		ParametersAcceptor $parametersAcceptor,
 		Scope $scope,
 		bool $isBuiltin,
-		$funcCall,
+		Expr\CallLike $funcCall,
```

Comment:
Please use exactly the union type in the PHPDoc. CallLike is wider than that - there are more classes there that are not supposed to be passed into FunctionCallParametersCheck.